network/retrieval/retrieve_test.go

```diff
@@ -183,7 +183,7 @@ func TestDeliveryForwarding(t *testing.T) {
 func setupTestDeliveryForwardingSimulation(t *testing.T) (sim *simulation.Simulation, uploader, forwarder, fetching enode.ID) {
 	sim = simulation.NewBzzInProc(map[string]simulation.ServiceFunc{
 		"bzz-retrieve": newBzzRetrieveWithLocalstore,
-	})
+	}, false)
```

Comment:
disableAutoConnect should be true in these tests i think no?

Reply:
That was the default value for HiveParams.DisableAutoConnect (implicit default false) as it was not set explicitly earlier. I agree that it should be true.